src/hwdb/hwdb.c

```diff
@@ -109,8 +114,9 @@ static int parse_argv(int argc, char *argv[]) {
 
 static int hwdb_main(int argc, char *argv[]) {
         static const Verb verbs[] = {
-                { "update", 1, 1, 0, verb_update },
-                { "query",  2, 2, 0, verb_query  },
+                { "update", 1,        1,        0,            verb_update },
+                { "query",  2,        2,        0,            verb_query  },
+                { "help",   VERB_ANY, VERB_ANY, VERB_DEFAULT, verb_help   },
```

Comment:
Also this one. Please add a test case in testsuite-17.10.sh

Reply:
ok